test/request-response-samples/Dummy__should_return_array_despite_maxOccurs_1/common.xsd

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:tns="http://www.Dummy.com/Common/Types" xmlns:xs="http://www.w3.org/2001/XMLSchema" targetNamespace="http://www.Dummy.com/Common/Types" elementFormDefault="unqualified" attributeFormDefault="unqualified">
+	<xs:complexType name="DummyResult">
+		<xs:sequence>
+			<xs:element name="DummyList" type="tns:DummyList" minOccurs="0"/>
+		</xs:sequence>
+		<xs:attribute name="code" type="xs:string" use="optional"/>
+	</xs:complexType>
+	<xs:complexType name="Dummy">
+		<xs:simpleContent>
+			<xs:extension base="xs:string">
+				<xs:attribute name="language" type="xs:language" use="optional"/>
+			</xs:extension>
+		</xs:simpleContent>
+	</xs:complexType>
+	<xs:complexType name="DummyList">
+		<xs:sequence>
+			<xs:element name="DummyElement" type="tns:Dummy"/>
```

Comment:
Why should a list with maximum length 1 considered an array ?

Reply:
For the reasons described in the issue that this PR seeks to resolve. See that issue for further explanation.

Reply:
Because it is a list. See issue for further discussion on this.

Reply:
This is existing behaviour found in strong-soap. Altering behaviour to honour maxOccurs=1 is outside the scope of this PR; doing so may actually break existing clients that rely on the dishonouring of maxOccurs=1.